modules/troubleshooting-disabling-autoreboot-mco-cli.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco-cli_{context}"]
+= Disabling Machine Config Operator from automatically rebooting by using the CLI
+
+To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can use the CLI to modify the machine config pool (MCP) to prevent automatic rebooting after the Operator makes changes to the machine config.
+
+[NOTE]
+====
+Pausing a machine config pool stops all system reboot processes and all configuration changes from being applied.
```

Comment:
This is also not entirely the case. Other operators today can cause reboots.

Reply:
@yuqi-zhang 
What if we put [your warning above](https://github.com/openshift/openshift-docs/pull/28440#discussion_r595529469) in place of this note. I think it conveys the same message, more or less.